### PR TITLE
Time build components

### PR DESCRIPTION
**Pull request by itziakos**
This changes travis.yml so that we can have a better idea on what it taking to much time.

It looks that the problematic items are:

- make lammps
- make simphony
- make simphony-openfoam
- make test-kratos

### CODE REVIEW

**Comment by itziakos** [comment]
@nathanfranklin is this ok to merge?

File: .travis.yml [hunk 1]
```diff
@@ -24,10 +24,17 @@ before_install:
...
+  - make numerrin
   - source ~/simphony/bin/activate
   - source /opt/openfoam222/etc/bashrc
   - make simphony
-  - make simphony-plugins
+  - make simphony-jyu-lb
+  - make simphony-lammps
+  - make simphony-mayavi
+  - make simphony-openfoam
+  - make simphony-kratos
```
Comment (nathanfranklin) [inline .travis.yml]: make simphony-numerrin

**Comment by nathanfranklin** [comment]
Just left one comment.  Otherwise, Looks good to me. :+1:

File: .travis.yml (hunk 1 above)
Comment (itziakos) [inline .travis.yml]: in travis-ci we cannot use `nummerin`.

Comment (nathanfranklin) [inline .travis.yml]: True.  We can still install it with `make simphony-numerri` but yeah, the tests would be skipped.

Comment (itziakos) [inline .travis.yml]: Hmm. we can at least test the install process. Let me add it

Comment (nathanfranklin) [inline .travis.yml]: Looks good as it is then

**Pull request merged**